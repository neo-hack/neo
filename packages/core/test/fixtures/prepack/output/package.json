{
  "name": "prepack",
  "version": "1.0.0",
  "description": "",
  "license": "ISC",
  "author": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "ci:publish": "pnpm run build && pnpm changeset publish",
    "ci:version": "pnpm changeset version",
    "ci:snapshot": "pnpm changeset version --snapshot beta",
    "ci:prerelease": "pnpm run build && pnpm changeset publish --tag beta",
    "lint:fix": "eslint . --fix",
    "prepare": "husky install"
  },
  "lint-staged": {
    "**/**/*.{js,ts,tsx,vue,json,yml,yaml}": ["eslint --fix"]
  },
  "devDependencies": {
    "@aiou/eslint-config": "^0.10.0",
    "@changesets/cli": "^2.26.0",
    "comparedir-test": "^1.0.1",
    "eslint": "^8.32.0",
    "husky": "^8.0.3",
    "lint-staged": "^13.1.0",
    "typescript": "^4.6.4"
  }
}