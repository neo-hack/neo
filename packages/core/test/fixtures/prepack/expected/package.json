{
  "name": "prepack",
  "version": "1.0.0",
  "description": "",
  "license": "ISC",
  "author": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "ci:publish": "pnpm run build && pnpm changeset publish",
    "ci:version": "pnpm changeset version",
    "ci:snapshot": "pnpm changeset version --snapshot snapshot",
    "ci:prerelease": "pnpm run build && pnpm changeset publish --no-git-tag --snapshot",
    "lint:fix": "eslint . --fix",
    "prepare": "husky install"
  },
  "lint-staged": {
    "**/**/*.{js,ts,tsx,json,yml,yaml,md}": ["eslint --fix"]
  },
  "devDependencies": {
    "@aiou/eslint-config": "^1.2.1",
    "@changesets/cli": "^2.26.0",
    "comparedir-test": "^1.0.1",
    "eslint": "^8.56.0",
    "husky": "^8.0.3",
    "lint-staged": "^13.1.0",
    "typescript": "^4.6.4",
    "cz-emoji": "^1.3.1"
  }
}